server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -981,8 +992,8 @@ public void test_copy_from_with_fail_fast_property_can_kill_all_nodes_with_failu
                 nodeIds.size());
             URI passingUri = generateURIToBeConsumedByProvidedReaderNumber(
                 target,
-                Arrays.asList("{\"a\":123456789},".repeat(10000).split(",")),
-                (readerNumber+1) % nodeIds.size(), // even with #nodes > 2, it does not matter who gets the passing uri.
+                Arrays.asList("{\"a\":123456789},".repeat(20000).split(",")),
```

Comment:
Why the increase of rows?

Reply:
sometimes the copy stmt on a node finished before receiving the fail_fast signal.